### DISCUSSION TITLE
Problems using git lfs and github

### DISCUSSION THREAD

**Discussion by lem0ncake** [community/community#47237] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi. I have been unable to push commit to a remote GitHub server for several days. A file weighing more than 100 Mb appeared in my project, I added lfs to the project and added the file to the tracklist, but now when I try to git push, I get the following message:

Remote "origin" does not support the Git LFS locking API. Consider disabling it with:
  $ git config lfs.https://github.com/lem0ncake/EnergyGames.git/info/lfs.locksverify false
[git@github.com](mailto:git@github.com): Permission denied (publickey).: exit status 255
error: failed to push some refs to 'github.com:lem0ncake/EnergyGames.git'

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lem0ncake ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
